site/index.html

```diff
@@ -85,10 +85,10 @@ <h2>BootstrapCDN</h2>
         </div>
         <h2>Bootstrap Icons</h2>
         <p class="lead font-weight-normal">
-          For the first time ever, Bootstrap has its own open source SVG icon library, designed to work best with our components and documentation.
+          For the first time ever, Bootstrap has its own open-source SVG icon library, designed to work best with our components and documentation.
```

Comment:
Remove the dash here, too.

```suggestion
          For the first time ever, Bootstrap has its own open source SVG icon library, designed to work best with our components and documentation.
```